projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { useRef } from 'react'
-import { StyleSheet } from 'react-native'
+import { StyleSheet, Share, Platform } from 'react-native'
```

Comment:
Am assssssuming, this is causing the red cross in the PR as `Platform` appears to be unused ...